### PR TITLE
ISPN-5717 Notify continuous query also when entry expires

### PR DESCRIPTION
**Pull request by vjuranek**
https://issues.jboss.org/browse/ISPN-5717

### CODE REVIEW

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/RemoteContinuousQueryTest.java [hunk 1]
```diff
@@ -151,6 +152,25 @@ public void resultLeaving(Object key) {
 
       expectElementsInQueue(joined, 0);
       expectElementsInQueue(left, 3);
+      
+      
+      remoteCache.clear();
+      user1.setAge(21);
+      user2.setAge(22);
+      remoteCache.put("expitedUser1", user1, 5, TimeUnit.MILLISECONDS);
```
Comment (anistor) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/RemoteContinuousQueryTest.java]: I think you meant expiredUser1

File: query/src/test/java/org/infinispan/query/continuous/ContinuousQueryTest.java [hunk 2]
```diff
@@ -23,14 +28,25 @@
 @Test(groups = "functional", testName = "query.continuous.ContinuousQueryTest")
 public class ContinuousQueryTest extends SingleCacheManagerTest {
 
+   protected ControlledTimeService timeService = new ControlledTimeService(0);
+   protected ExpirationManager manager;
+   
+   @Override
+   protected void setup() throws Exception {
+      super.setup();
+      manager = TestingUtil.extractComponent(cache, ExpirationManager.class);
```
Comment (anistor) [inline query/src/test/java/org/infinispan/query/continuous/ContinuousQueryTest.java]: It's simpler to just do ```cache.getAdvancedCache().getExpirationManager()```

**Comment by anistor** [comment]
Looks good. I'll fix these two minor comments myself and will pull it.

**Comment by vjuranek** [comment]
thanks @anistor !

**Comment by anistor** [comment]
Integrated in master. Thanks @vjuranek !

**Pull request closed without merging**